compiler/rustc_lint/src/context.rs

```diff
@@ -336,6 +337,20 @@ impl LintStore {
         }
     }
 
+    /// True if this symbol represents a lint group name.
+    pub fn is_lint_group(&self, lint_name: Symbol) -> bool {
+        debug!(
+            "is_lint_group(lint_name={:?}, lint_groups={:?})",
+            lint_name,
+            self.lint_groups.keys().collect::<Vec<_>>()
+        );
+        let lint_name_str = &*lint_name.as_str();
+        self.lint_groups.contains_key(&lint_name_str) || {
+            let warnings_name_str = crate::WARNINGS.name_lower();
```

Comment:
For some reason, `WARNINGS` is not in this list.

Reply:
Hmm. Are there any other "obvious" groups that might also be not on that list for some reason?

Reply:
No. The reason for WARNINGS is that it is mega hardcoded in the way that it's implemented; it's not a real lint group.